solutions/gold/usaco-993.mdx

```diff
@@ -22,48 +32,54 @@ We only need to keep track of the number of days that Bessie has been traveling
 #include <bits/stdc++.h>
 using namespace std;
 
-const int MAX_DAYS = 1000;
-
 int main() {
-	freopen("time.in", "r", stdin);
-	freopen("time.out", "w", stdout);
+	ifstream cin("time.in");
+	ofstream cout("time.out");
 
 	int n, m, c;
 	cin >> n >> m >> c;
 
-	vector<int> earn(n);
-	for (int i = 0; i < n; i++) { cin >> earn[i]; }
+	vector<int> moonies(n + 1);
```

Comment:
typically in usaco guide we zero index unless it's a fair bit more annoying to write than with 1 indexing